cfe_internal/update/update_policy.cf

```diff
@@ -453,7 +453,7 @@ body perms u_mo(p,o)
 # @param p Desired file owner (username or uid)
 {
       mode   => "$(p)";
-      !windows::
+      !windows.!termux::
         owners => {"$(o)"};
```

Comment:
Not windows AND not Termux don't we want to skip it if windows or termux is defined?

How about `!(windows|termux)::` instead? I think it reads more clearly

Reply:
sure.